### PR TITLE
Add coincheck

### PR DESCRIPTION
**Pull request by cc-ShuheiSawamura**
Coincheck(https://coincheck.com) is one of the largest crypto exchange in Japan. Alexa rank of the exchange is 14977 globally and 1225 in Japan. 

https://www.alexa.com/siteinfo/coincheck.com

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -239,6 +239,13 @@ websites:
       - u2f
     doc: https://support.coinbase.com/customer/en/portal/articles/1658338
 
+  - name: Coincheck
+    url: https://coincheck.com/
+    img: coincheck.png
+    tfa:
+      - totp
+    doc: https://faq.coincheck.com/s/article/20204
```
Comment (phallobst) [inline _data/cryptocurrencies.yml]:
```suggestion
    doc: https://faq.coincheck.com/s/article/20204?language=en_US
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 1 above)
Comment (cc-ShuheiSawamura) [reply _data/cryptocurrencies.yml]: Thanks phallobst. I committed your suggestion.

**Comment by phallobst** [comment]
Could you pass the icon through tinypng.com to shrink its file size, please?

**Comment by cc-ShuheiSawamura** [comment]
> Could you pass the icon through tinypng.com to shrink its file size, please?

I changed the picture to the one shrinked by tinypng.com.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**